### PR TITLE
chore: add js restify notification template

### PR DESCRIPTION
**Pull request by a1exwang**
tested locally
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/13791043

### CODE REVIEW

**Comment by a1exwang** [comment]
E2E TEST: n/a

**Review by swatDong — APPROVED** [review]

File: templates/bot/js/notification-restify/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "echobot",
```
Comment (yukun-dong) [inline templates/bot/js/notification-restify/package.json:2]:
```suggestion
  "name": "notification-restify-bot",
```
to be consistent with other templates

File: templates/bot/js/notification-restify/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "version": "1.0.0",
+  "description": "Microsoft Teams Toolkit hello world Bot sample",
```
Comment (yukun-dong) [inline templates/bot/js/notification-restify/package.json:4]: same as above

File: templates/bot/js/notification-restify/package.json (hunk 1 above)
Comment (a1exwang) [reply templates/bot/js/notification-restify/package.json]: fixed

File: templates/bot/js/notification-restify/package.json (hunk 2 above)
Comment (a1exwang) [reply templates/bot/js/notification-restify/package.json]: fixed

**Review by yukun-dong — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**